pkg/models/user.go

```diff
@@ -13,7 +15,45 @@ type UserData struct {
 }
 
 type UserPassword struct {
-	Password string `json:"password" bson:",omitempty" validate:"required,min=5,max=30"`
+	// Plain contains the plain text password.
+	Plain string `json:"password" bson:"-" validate:"required,password"`
+	// Password contains the hashed pasword from plain text.
+	Password string `json:"-" bson:"password"`
```

Comment:
Isn't it a good idea to call this 'Hash' instead of 'Password'?

Reply:
`HashedPassword`?

Reply:
Done. It is now `PlainPassword` and `HashedPassword`.